### DISCUSSION TITLE
Branch(es) in repository have disappeared

### DISCUSSION THREAD

**Discussion by Trimatix** [community/community#27482] [Other Feature Feedback, Questions, & Ideas]
Hi all,
One of my repositories certainly used to have more than one branch. It has been a short while since i have used this repository, so I don't remember how many there were, but there was certainly at the very least master and dev. My local copy of the repository also has three others.

When I went to contribute today, I saw that **only** the master branch remained. I am **certain** that I have not deleted my branches, **certainly** not dev, where all of my work from the last **14 months** was living!

How could this have happened? How can I recover this huge volume of work?

The repository: https://github.com/Trimatix/BASED

**Comment by airtower-luna** [comment]
> My local copy of the repository also has three others.

If you still have a local copy, the simplest solution is to push the branch from there.

Other than that I'm baffled, I've never seen branches just disappear. :open_mouth: If you're sure it has been pushed before you can try to [contact support](https://support.github.com/request) and see if they can figure out what happened, and if they can restore your data from a backup.
